cypress/integration/chat_spec.js

```diff
@@ -0,0 +1,25 @@
+const url = Cypress.config().baseUrl;
+
+describe('on send a message', () => {
+    beforeEach(() => {
+        cy.login('Bob');
+        cy.sendMessage();   
+    });
+
+    it('displays the message', () => {     
+        cy.get('[data-cy="messages"]').should('have.text', 'Bob : message');
+    });
+});
```

Comment:
Another test might be making sure that the input clears when the message is sent?